uaclient/testing/helpers.py

```diff
@@ -1,12 +1,17 @@
-from contextlib import contextmanager
+try:  # Drop try-except after xenial EOL
+    from contextlib import AbstractContextManager
+except ImportError:
+    AbstractContextManager = object
```

Comment:
I'm okay with these changes but I'm not sure if they're necessary. `tox -e py35-test` is already passing without these changes. Were you running into failures?

Reply:
Yes, I run into failures in py-35 because [AbstractContextManager](https://docs.python.org/3/library/contextlib.html#contextlib.AbstractContextManager) was introduced in py-36.